tests/README.md

```diff
@@ -0,0 +1,77 @@
+## Backend development
+
+The backend test suite can be run locally. 
+
+### Start the services
+
+We need all but the main `backend` service running.  Since multiple `backend` instances are allowed, we will simply start everything up
+
+    # ENSURE THE IMAGES ARE CLEAN
+    docker-compose down
+    # SETUP ALL IMAGES
+    docker-compose up --build
+
+### Install into virtual environment
+
+Treeherder, Django, and the required libraries will ruin you local Python install. It is best to setup a virtual environment to contain the quirks it introduces.
+
+Be sure you are in the `treeherder` main directory
+
+    pip install virtualenv
```

Comment:
There's also a package called ``virtualenvwrapper`` that has some nicer commands like ``workon`` instead of ``source venv/bin/activate``.  Just mentioning.  No need to change this.

Reply:
I didn't want to bring it up earlier to avoid derailing this PR, however, if we're going to use tools to manage Python virtual environments we should go directly to [poetry](https://github.com/sdispater/poetry) since it is a modern solution that @ahal and I are recommending. Among many features it manages virtualenvs & activation for you.

`virtualenv` and `virtualenvwrapper` are old ways of doing things.

Reply:
So many virtual environment managers to choose from!